reference-architectures/reference-architecture-ccs.md

```diff
@@ -0,0 +1,157 @@
+---
+
+copyright:
+  years: 2024
+lastupdated: "2024-06-12"
+
+subcollection: security-services
+
+authors:
+- name: Dharmesh Bhakta
+  email: bhakta@ibm.com
+
+# The release that the reference architecture describes
+version: 1.0
+
+# Use if the reference architecture has deployable code.
+# Value is the URL to land the user in the IBM Cloud catalog details page for the deployable architecture.
+# See https://test.cloud.ibm.com/docs/get-coding?topic=get-coding-deploy-button
+deployment-url: <url>
```

Comment:
As soon as the stack is onboarded to the catalog at cloud.ibm.com/catalog, you can update this URL.